### DISCUSSION TITLE
UAT Site Now Live

### DISCUSSION THREAD

**Discussion by Galope-Jhean** [PUP-BSIT/project-vrzn_appdev#35] [General]
Hello team. 

[UAT](https://uat.spaceshare.site/home) site is now live, please test our `Authentication Module`. 

This includes:
 - `Sign Up`
 - `Registration`
 - `Email Verification`
 - `Sign Out`

 Need the results before Tuesday.

Tagging @go-christianharrel  as well for updates.

Thanks.

**Comment by Naoe-Adrian** [comment]
this is noted

**Comment by Dimasacat-Andronicus** [comment]
Acknowledged, I'll handle it.

**Comment by go-christianharrel** [comment]
@Dimasacat-Andronicus use a spreadsheet for this task so we can integrate it with UiPath automation. We'll prioritize working with the spreadsheet first, then move on to Xray Test Management afterward. Thank you, and happy testing! :rocket: 

**UPDATE: We will no longer be using Xray.**

**Comment by Dimasacat-Andronicus** [comment]
Noted, currently working on it.

**Comment by Dimasacat-Andronicus** [comment]
- You can see the Test case for June 09, 2024 in documents branch under "document/quality_assurance/test_case". 
https://github.com/PUP-BSIT/project-vrzn_appdev/blob/documents/document/quality_assurance/test_case/Test_case-june_09_2024.pdf

- For more concerns, don't hesitate to reach me out.

**Reply by go-christianharrel** [reply]
Create an issue from this discussion @Dimasacat-Andronicus regarding the [Authentication Module](https://github.com/PUP-BSIT/project-vrzn_appdev/discussions/35?converting=1#discussion-6802664) :rocket:

**Reply by Dimasacat-Andronicus** [reply]
This is noted, will do that later.
